src/OpenRoad.tcl

```diff
@@ -426,8 +441,12 @@ proc place_inst { args } {
     utl::error ORD 65 "Cannot create instance $inst_name of $cell_name."
   }
 
-  $inst setOrigin $x $y
   $inst setOrient $orient
+  if { [info exists keys(-origin)] } {
+    $inst setOrigin $x $y
```

Comment:
setOrient & setOrigin are not commutative

Reply:
What do you mean? Looking at the code for these two functions it doesn't look like either depends on the other. setLocation uses Origin which means order matters for that.

Reply:
You are correct.